samples/CarterSample/Startup.cs

```diff
@@ -26,37 +32,16 @@ public void Configure(IApplicationBuilder app, IConfiguration config)
             config.Bind(appconfig);
```

Comment:
Configuration isn't being used here.

Reply:
Rejigged Configuration in the sample to use the new options approach in newer commit